### PR TITLE
[I18N] estate: add translations for `es_MX` T#69469

### PR DESCRIPTION
**Pull request by Suavesito-Olimpiada**
In this commit, translations to `es_MX` were added, following the specification of the section "Translating Modules" of the technical training.

This is ready for review @deivislaya.

### CODE REVIEW

**Comment by Suavesito-Olimpiada** [comment]
Ready to merge. @deivislaya

File: estate/i18n/es_MX.po [hunk 1]
```diff
@@ -0,0 +1,770 @@
...
+#. module: estate
+#. odoo-python
+#: code:addons/estate/models/estate_property_offer.py:0
+#, python-format
+msgid "Property cannot refuse offers."
+msgstr "La proppiedad no puede rechazar ofertas."
+
+#. module: estate
+#: model_terms:ir.ui.view,arch_db:estate.estate_property_tag_view_form
+msgid "Property tags"
```
Comment (deivislaya) [inline estate/i18n/es_MX.po]:
```suggestion
msgid "Property Tags"
```

File: estate/i18n/es_MX.po [hunk 2]
```diff
@@ -0,0 +1,770 @@
...
+
+#. module: estate
+#: model:ir.model.fields.selection,name:estate.selection__estate_property_offer__status__refused
+msgid "Refused"
+msgstr "Rechazada"
+
+#. module: estate
+#: model_terms:ir.ui.view,arch_db:estate.estate_property_view_form
+msgid "SOLD"
+msgstr "VENDIDA"
```
Comment (deivislaya) [inline estate/i18n/es_MX.po]:
```suggestion
msgid "Sold"
msgstr "Vendida"
```

File: estate/i18n/es_MX.po [hunk 3]
```diff
@@ -0,0 +1,770 @@
...
+#: model:ir.model.fields,help:estate.field_estate_property__date_availability
+msgid "The property is available starting this date."
+msgstr "La propiedad será disponible empezando esta fecha."
+
+#. module: estate
+#. odoo-python
+#: code:addons/estate/models/estate_property.py:0
+#, python-format
+msgid "The selling price cannot be lower than 90% of expected price."
+msgstr "El precio de venta no puede ser menor que el 90% del precio esperado"
```
Comment (deivislaya) [inline estate/i18n/es_MX.po]:
```suggestion
msgstr "El precio de venta no puede ser menor que el 90% del precio esperado."
```

**Comment by Suavesito-Olimpiada** [comment]
@deivislaya issues solved.

**Pull request merged**